test/HatsModule.t.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import { Test, console2 } from "forge-std/Test.sol";
+import { HatsModule, IHats } from "../src/HatsModule.sol";
+
+contract HatsModuleTest is Test {
+  event HatsModuleHarness_SetUp(bytes initData);
+
+  string public MODULE_VERSION = "test module";
+  address hatAddress = makeAddr("this is a hat address");
+  uint256 hatId = 1;
+  HatsModule public hatsModule;
+  bytes largeBytes =
+    abi.encodePacked("This is a fairly large bytes object, but is it really? Yes it is, it's quite large");
+  uint256 public largeBytesLength = largeBytes.length;
+
+  function _deployModule(string memory _moduleVersion, address _hatAddress, uint256 _hatId) public virtual {
+    // super.setUp();
```

Comment:
This can probably be commented out.

Reply:
Done: https://github.com/alexkeating/hats-module/pull/3/commits/9caaf8ba495f4bee16257e8b0d82f808dcb7d229